extensions/amp-story/1.0/test/test-amp-story.js

```diff
@@ -279,8 +279,8 @@ describes.realWin('amp-story', {
         });
 
     const eventObj = createEvent('keydown');
-    eventObj.keyCode = KeyCodes.RIGHT_ARROW;
-    eventObj.which = KeyCodes.RIGHT_ARROW;
+    eventObj.key = Keys.RIGHT_ARROW;
+    eventObj.which = Keys.RIGHT_ARROW;
```

Comment:
I am not sure why this code mocks the `which` property, but the spec defines it to be a specifically numerical value. https://www.w3.org/TR/uievents/#dom-keyboardevent-which The test code should either remove `which` if it is not needed, or if it is needed, the value should conform to the spec

Reply:
Turns out `keyCode` and `which` might differ on some browsers. https://unixpapa.com/js/key.html 

I thought it safest to keep the code check to numeric here. 
Fixed in the latest patch.